powerfulseal/policy/scenario.py

```diff
@@ -57,7 +57,11 @@ def execute(self):
         for step in steps:
             for action_name, action_method in self.action_mapping.items():
                 if action_name in step:
-                    ret = action_method(schema=step.get(action_name))
+                    step_action = step.get(action_name)
+                    if "retries" in step_action.keys():
```

Comment:
The retry method seems to have a pass-through, meaning that this check is redundant. Probably best to only do it in one place.

Reply:
Updated